kibitzr/app.py

```diff
@@ -137,13 +137,26 @@ def schedule_checks(checkers):
         schedule.clear()
         for checker in checkers:
             conf = checker.conf
-            period = conf["period"]
-            logger.info(
-                "Scheduling checks for %r every %r seconds",
-                conf["name"],
-                period,
-            )
-            schedule.every(period).seconds.do(checker.check)
+            for s in conf['schedule']:
+                job = getattr(schedule.every(s['interval']), s['unit'])
```

Comment:
I think we need to add validation for s['unit'] to be one of the schedule's allowed values in `conf.py`.
Also please add a test for an invalid unit, like `month` or `year`. It should check that `ConfigurationError` is raised.

Reply:
I though about validating unit but choose not to so that kibitzr 'unit' is whatever 'unit' is valid for 'schedule', now and in the future without the need to keep a list of possible values updated.
I could put in a try-catch to raise ConfigurationError when unit or interval are invalid.

Reply:
I don't mean to copy-paste available units from the schedule, but rather call something like `getattr(schedule.every(1), s['unit'])` during configuration parsing.

Reply:
great, will do.